app/src/main/java/com/willowtree/vocable/presets/PresetsRepository.kt

```diff
@@ -0,0 +1,37 @@
+package com.willowtree.vocable.presets
+
+import android.content.Context
+import com.willowtree.vocable.room.Category
+import com.willowtree.vocable.room.Phrase
+import com.willowtree.vocable.room.VocableDatabase
+import org.koin.core.KoinComponent
+import org.koin.core.get
+
+class PresetsRepository(context: Context) : KoinComponent {
+
+    private val database = VocableDatabase.getVocableDatabase(context)
```

Comment:
Why don't you set up a Koin module for your DB and inject it into your repository class?

Reply:
👍 We're going to look into this once we get the saving presets functionality working.